pkg/device/device.go

```diff
@@ -0,0 +1,41 @@
+package device
```

Comment:
I still think it's cleaner to use the macros. Then you won't be responsible if it _ever_ changes. Though this macro is a simple one. 

I would just add build tags for now.